manta-js/examples/asset-webpack-ts/index.ts

```diff
@@ -124,9 +124,12 @@ const toPrivateOnlySignTest = async () => {
     const initalPrivateBalance = await privateWallet.getPrivateBalance(assetId);
     console.log("The inital private balance is: ", initalPrivateBalance.toString());
 
-    const signResult = await privateWallet.toPrivateSend(assetId, amount, polkadotConfig.polkadotSigner, polkadotConfig.polkadotAddress);
+    const signResult = await privateWallet.toPrivateBuild(assetId, amount, polkadotConfig.polkadotSigner, polkadotConfig.polkadotAddress);
 
-    console.log("The result of the signing: ", signResult);
+    console.log("The result of the signing: \"0x", JSON.stringify(signResult));
```

Comment:
trailling " will be missing no?

Reply:
Yeah, need to remove this extra "0x alltogether.

Reply:
what i mean is the `signResult` contains `0xaaabbcc.."` including a trailing `"` if i understood you correctly.

you remove the leading `\"` but not the trailing one. This could lead to an invalid string/string with extra `"` char being handed out

Reply:
Im not changing this code it remains `console.log("The result of the signing: ", signResult);`